pkg/oc/bootstrap/docker/up.go

```diff
@@ -797,34 +803,22 @@ func (c *CommonStartConfig) CheckDockerInsecureRegistry(out io.Writer) error {
 
 // CheckNsenterMounter checks whether the Docker host can use the nsenter mounter from Kubernetes.
 // Otherwise, a shared volume is needed in Docker
-func (c *CommonStartConfig) CheckNsenterMounter(out io.Writer) error {
-	useNsenterMount, err := c.HostHelper().CanUseNsenterMounter()
-	if err != nil {
-		return err
-	}
-	if useNsenterMount && c.isRHDocker {
-		c.UseNsenterMount = true
-		fmt.Fprintf(out, "Using nsenter mounter for OpenShift volumes\n")
-	} else {
-		c.UseNsenterMount = false
-		fmt.Fprintf(out, "Using Docker shared volumes for OpenShift volumes\n")
-	}
-	return nil
+func (c *CommonStartConfig) CheckNsenterMounter(out io.Writer) (bool, error) {
+	return c.HostHelper().CanUseNsenterMounter()
 }
 
 // CheckDockerVersion checks that the appropriate Docker version is installed based on whether we are using the nsenter mounter
 // or shared volumes for OpenShift
 func (c *CommonStartConfig) CheckDockerVersion(out io.Writer) error {
```

Comment:
directly embed this logic in the `Check` method now?  It no longer mutates state, is only called in one location, and since it is writing it doesn't read like a library method.